pkg/graph/graph.go

```diff
@@ -125,15 +123,15 @@ func (g *Graph) generateCommon() {
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Failed to set to digraph: %v\n", err)
 	}
-	err = g.gviz.SetName("G")
```

Comment:
Let's keep top level graph `"G"` and add multiple subgraphs for each namespace inside it.